### ISSUE TITLE
:'<,'>Hammer

### ISSUE THREAD

**Issue by trapd00r** [wikimatze/hammer.vim#25]
Would it be possible to allow for Hammer to take a visual selection and apply its magic only on that instead of the whole buffer? Related to #24, I tried to visually select the POD part and :Hammer it, with no success. I think it'd be useful.

**Comment by wikimatze** [comment]
Hi @trapd00r, I'm thinking to implement this feature for Markdown files at first - if this is working, maybe then I can help you with POD. But you have to help me with Perl and Pod then, I'm not familiar with it and may need your help.

**Comment by trapd00r** [comment]
Hi @matthias-guenther,

sure thing, just ping me when I'm needed!

**Comment by ghost** [comment]
Why are you implementing this feature per-filetype?

**Comment by wikimatze** [comment]
Yeah I will, but first I will try this for markdown and if that works, I will ping @trapd00r to test this for Perl.
